source/agora/node/Registry.d

```diff
@@ -666,7 +687,7 @@ private struct TypedPayload
 }
 
 /// Contains infos related to either `validators` or `flash`
-private struct ZoneData
+private struct ZoneData (string TemplateTableName = "")
```

Comment:
In this scenario, I don't think a default value is mandated. You usually *want* the user to provide a default value, because not doing so is most likely to be a bug than not. But then again, see the other notes about templating based on a value.